.github/workflows/new-build-core-template.yml

```diff
@@ -208,6 +211,14 @@ jobs:
           path: |
             ./contracts
 
+      # TODO Remove it when we migrate to foundry inside contracts repository
+      - name: Create necessary artefacts
+        shell: bash
+        run: |
+          mkdir -p contracts/l1-contracts/artifacts/
```

Comment:
nit: you can create all the directories with single command:
`mkdir -p contracts/l1-contracts/artifacts contracts/l1-contracts/out`